mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -98,7 +98,7 @@ class GBTClassifier @Since("1.4.0") (
    * E.g. 10 means that the cache will get checkpointed every 10 iterations.
    * This is only used if cacheNodeIds is true and if the checkpoint directory is set in
    * [[org.apache.spark.SparkContext]].
-   * Must be >= 1.
+   * Must be &gt;= 1.
```

Comment:
I believe it should be {@literal >}. This would not be printed as `>` but `&gt` in javadoc IIRC.